pkg/daemon/ceph/cleanup/disk.go

```diff
@@ -145,23 +145,42 @@ func (s *DiskSanitizer) returnPVDevice(disk string) []string {
 	return strings.Split(output, ":")
 }
 
-func (s *DiskSanitizer) buildDDArgs(disk string) []string {
-	ddArgs := []string{
-		fmt.Sprintf("if=%s", ddIf),
-		fmt.Sprintf("of=%s", disk),
-		fmt.Sprintf("bs=%s", ddBS),
-		fmt.Sprintf("count=%s", ddCount),
-		fmt.Sprintf("oflag=%s", ddFlags),
+func (s *DiskSanitizer) buildDataSource() string {
+	return fmt.Sprintf("/dev/%s", s.sanitizeDisksSpec.DataSource.String())
+}
+
+func (s *DiskSanitizer) buildShredArgs(disk string) []string {
+	var shredArgs []string
+
+	// If data source is not zero, then let's add zeros at the end of the pass
```

Comment:
I'm confused by this option. If the source is not zero, we want to append `--zero`? Seems backwards.

Reply:
If the data source is not zero it's random, then shred will do another pass of zeros once done. This removes any indication that shredding was done :)